src/test/resources/services-local.conf

```diff
@@ -28,11 +28,18 @@
 # Example 2: (the base url for helloworld-service is http://localhost:9000)
 # helloworld-service.port = 9000
 
-
 services {
-    auth-login-stub{
-      protocol = http
-      host = localhost
-      port = 9949
-    }
+    auth-login-stub.port = 9949
+    economic-crime-levy-registration-frontend.port = 14000
+#      {
+#       protocol = http
+#       host = localhost
+#       port = 14000
+#     }
+#     economic-crime-levy-returns-frontend.port = 14002
+#     {
+#       protocol = http
+#       host = localhost
+#       port = 14002
+#     }
```

Comment:
did you mean to leave this commented out?

also is there a reason for moving away from the nested structure?